en/flight_modes/position_mc.md

```diff
@@ -27,7 +24,13 @@ The diagram below shows the mode behaviour visually (for a mode 2 transmitter).
 
 ## Technical Summary
 
-Roll controls left-right speed over ground, pitch controls front-back speed over ground, and throttle controls climb/descent speed (as in [Altitude](../flight_modes/altitude.md) mode). Yaw controls yaw *rate* over horizontal plane (as in [Manual/Stabilized](../flight_modes/manual_stabilized_mc.md) mode).  When roll, pitch and throttle inputs are all centered (inside deadzone) the multirotor will hold position (in x, y, z frame), compensating for drift due to wind (or other sources).
+Manual/Stabilized mode with <em>3D position stabilization</em> (centered sticks level vehicle and hold it to fixed position and altitude against wind).
+
+* Centered RPT sticks (in RC deadzone) hold x, y, z position steady against any wind and levels attitude.
+* Outside center:
+  * Roll/Pitch sticks control tilt angle in those orientations, resulting in corresponding left-right and forward-back movement.
+  * Throttle stick controls up/down speed (and movement speed in other axes).

```

Comment:
`(and movement speed in other axes)` If I understand this correct, it's not right: throttle really only controls up/down speed, horizontal is not affected.
You could mention that centering the throttle stick will make the vehicle keep the altitude.